### ISSUE TITLE
Epoch information is lacking pulse high times

### ISSUE THREAD

**Issue by t-b** [AllenInstitute/MIES#805]
The epoch info of a stimset with 5 pulse train pulses looks like

```
0.0000000,0.0200000,Inserted TP;Test Pulse;,0,
0.0000000,0.0050000,Baseline,1,
0.0050000,0.0150000,Inserted TP;Test Pulse;pulse;Amplitude=10;,1,
0.0150000,0.0200000,Baseline,1,
0.0200000,0.4300050,Stimset,0,:0.0200000,0.4300050,Epoch=0;Type=Pulse Train;Amplitude=1;,1,
0.0200000,0.1200000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=0;,2,
0.1200000,0.2200000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=1;,2,
0.2200000,0.3200000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=2;,2,
0.3200000,0.4200000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=3;,2,
0.4200000,0.4300050,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=4;,2,
```

And if we just look at the first pulse

```
0.0200000,0.1200000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=0;,2,:
```

![grafik](https://user-images.githubusercontent.com/1185677/105778859-81e0cf00-5f6d-11eb-845d-ac5aaa436987.png)

this reveals that we don't have the information about the pulse baseline as separate info. But we need that for the enhanced spike control.

So we need new entries like

```
0.0200000,0.0300000,Epoch=0;Type=Pulse Train;Amplitude=1;Pulse=0;Active,3,:
0.0300000,0.1200000,Epoch=0;Type=Pulse Train;Pulse=0;Baseline,3,:
```

**Comment by MichaelHuth** [comment]
Yes that would work, though we should chat about the details.

**Comment by t-b** [comment]
Implemented in #823
